test/function/samples/transform-source-function-gets-two-args/_config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+	description:
+		'pass the file path to sourcemapPathTransform',
+	options: {
+		output: {
+			name: 'myModule',
+			sourcemap: true,
+			file: path.resolve(__dirname, 'main.js'),
+			sourcemapPathTransform: (relativePath, sourcemapPath) => path.resolve(sourcemapPath, relativePath)
+		}
+	},
+	generateError: {
+		code: 'VALIDATION_ERROR',
+		message: 'sourcemapPathTransform function must return a string.'
+	}
```

Comment:
this section is wrong but I didn't find a way to assert about the sourcemap output or even just the js output... pointers here would be helpful.

Reply:
Why don't you just extend this one to use the new parameter: https://github.com/rollup/rollup/tree/master/test/sourcemaps/samples/transform-source-paths ? Alternatively, you can put a modified one next to it.